test/api/user/DELETE-user.test.js

```diff
@@ -27,10 +33,30 @@ describe('DELETE /user', () => {
   });
 
   context('user in solo group', () => {
-
-    it('deletes party when user is the only member');
-
-    it('deletes private guild when user is the only member');
+    let group;
+    it('deletes party when user is the only member', () => {
+      return expect(generateGroup(user, {
```

Comment:
I think it makes sense to pull this out into a before block, since it's less about testing the route and more about setting up to test the route. 

In that case, it probably makes sense to create two contexts, one for the guild and one for the party.